examples/virtio-snd/client_vmm.c

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2023, UNSW
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#include <stddef.h>
+#include <stdint.h>
+#include <microkit.h>
+#include "util/atomic.h"
+#include "util/util.h"
+#include "arch/aarch64/vgic/vgic.h"
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/au-ts/libvmm/commit/5c8773b6b2a131ee159d79874101cc07a498928e